src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -224,111 +265,116 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 						error(sender, "enable.all.io error", ExceptionUtils.toString(e));
 					}
 				} else {
-					File scriptFile = getScriptFromArgs(sender, args);
-					if (scriptFile == null)
+					File[] scriptFiles = getScriptsFromArgs(sender, args);
+					if (scriptFiles == null || scriptFiles.length == 0)
 						return true;
 
-					if (!scriptFile.isDirectory()) {
-						if (ScriptLoader.getLoadedScriptsFilter().accept(scriptFile)) {
-							info(sender, "enable.single.already enabled", scriptFile.getName(), StringUtils.join(args, " ", 1, args.length));
-							return true;
-						}
+					for (File scriptFile : scriptFiles) {
+						if (!scriptFile.isDirectory()) {
+							if (ScriptLoader.getLoadedScriptsFilter().accept(scriptFile)) {
+								info(sender, "enable.single.already enabled", scriptFile.getName(), StringUtils.join(args, " ", 1, args.length));
+								return true;
+							}
 
-						try {
-							scriptFile = toggleFile(scriptFile, true);
-						} catch (IOException e) {
-							error(sender, "enable.single.io error", scriptFile.getName().substring(ScriptLoader.DISABLED_SCRIPT_PREFIX_LENGTH), ExceptionUtils.toString(e));
-							return true;
-						}
+							try {
+								scriptFile = toggleFile(scriptFile, true);
+							} catch (IOException e) {
+								error(sender, "enable.single.io error", scriptFile.getName().substring(ScriptLoader.DISABLED_SCRIPT_PREFIX_LENGTH), ExceptionUtils.toString(e));
+								return true;
+							}
 
-						final String fileName = scriptFile.getName();
-						info(sender, "enable.single.enabling", fileName);
-						ScriptLoader.loadScripts(scriptFile, logHandler)
-							.thenAccept(scriptInfo -> {
-								if (logHandler.numErrors() == 0) {
-									info(sender, "enable.single.enabled", fileName);
-								} else {
-									error(sender, "enable.single.error", fileName, logHandler.numErrors());
-								}
-							});
-					} else {
-						Set<File> scriptFiles;
-						try {
-							scriptFiles = toggleFiles(scriptFile, true);
-						} catch (IOException e) {
-							error(sender, "enable.folder.io error", scriptFile.getName(), ExceptionUtils.toString(e));
-							return true;
-						}
+							final String fileName = scriptFile.getName();
+							info(sender, "enable.single.enabling", fileName);
+							ScriptLoader.loadScripts(scriptFile, logHandler)
```

Comment:
should we really be individually enabling every script, with feedback sent for every single one? Shouldn't this batch the enables together?